packages/affiliate/src/Affiliate.php

```diff
@@ -1,58 +1,47 @@
 <?php
```

Comment:
We'll need to keep the original file for compatibility reasons. You can an example on how to proceed with other classes that have been moved to packages.

Should we also add it to the compat package?